src/NewBookingForm.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+
+function NewBookingForm(props) {
+  const [bookingForm, setBookingForm] = useState("");
+  const [title, setTitle] = useState("");
+  const [firstName, setFirstname] = useState("");
+  const [surname, setSurname] = useState("");
+  const [email, setEmail] = useState("");
+  const [roomId, setRoomID] = useState("");
+  const [checkInDate, setCheckInDate] = useState("");
+  const [checkOutDate, setCheckOutDate] = useState("");
+  const [totalnumOfNights,setTotalnumOfNights] = useState("")
+
+  function handleBookingForm(event) {
+      if (event.target.name === "title") {
+      setTitle(event.target.value);
+      }else if (event.target.name === "firstName") {
+      setFirstname(event.target.value);
+    } else if (event.target.name === "surname") {
+      setSurname(event.target.value);
+    } else if (event.target.name === "email") {
+      setEmail(event.target.value);
+    } else if (event.target.name === "roomId") {
+      setRoomID(event.target.value);
+    } else if (event.target.name === "checkInDate") {
+      setCheckInDate(event.target.value);
+    }else if (event.target.name === "checkOutDate") {
+      setCheckOutDate(event.target.value);
+    }
+}
```

Comment:
Can we think of a way to simplify this? There is a way of doing this without all of the `if/elses`.
It would involve keeping them all in one state object 😄  If you are not sure, then reach out to me I can help!

Reply:
Soooooo......yes please show me how to make it simpler. I did it like this because it is easier for me to understand what I am doing and trying to achieve. But always happy to learn a new way of doing things. Thanks Sam.xxx

Reply:
This is a thing a lot of people have done, so I am going to bring this to the help center! 😄